src/utils/sqrt.js

```diff
@@ -0,0 +1,26 @@
+const MAX_DECIMAL = 7;
+
+const roundNotInteger = (numSqrt, result) => {
```

Comment:
you may add the third parameter `maxDecimal` and set the default value `MAX_DECIMAL`. Then your function will be more flexible